foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -339,10 +339,19 @@ AEPIdentityItem *item = [[AEPIdentityItem alloc] initWithId:@"user@example.com"
 
 ## resetIdentities
 
-Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\) . Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+Clears all identities stored in the Identity extension and generates a new Experience Cloud ID \(ECID\). Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+
+This is a destructive action. Once an ECID is removed it cannot be reused. The new ECID generated by this API can impact metrics like unique visitors and cause duplication of user profiles and device profiles. To remove existing custom identifiers, use [`removeIdentity`](#removeidentity).
+
+Some example use cases for this API are: 
+- During debugging, to see how new ECIDs (and other identifiers paired with it) behave with existing rules and metrics.
+- A last-resort reset for when an ECID should no longer be used.
+
+This API is not recommended for:
+- Standard app lifecycle events (bootup/foreground/background)
```

Comment:
We can move this one here: To remove existing custom identifiers, use [`removeIdentity`](#removeidentity).


This API is not recommended for:
...
- Removing existing customer identifiers; use the [`removeIdentity`](#removeidentity) API instead.

Reply:
I like this change! It definitely makes the not recommended use case clearer